Update zbll-practice.mdx (#47)
ericx20/zz-method-docs: pages/en/improvement-guide/ll/zbll-practice.mdx

- execution of the algorithm (which cover points 2 and 3).
- post AUF (which covers point 4).

> Now for the actual alg-learning part: By going into the ZBLL sheets you'll see the algorithms getting bundled together in groups of 12, by their corner permutation. It's important that you try to go through this group of 12 in one sitting. You don't want to learn only a few algorithms at once because that will hinder your ability to distinguish cases from each other, which will hinder your 'Case-Alg' connection, and make recognition of the cases harder down the line. On the other hand, you don't want to learn too much at once because you don't want to spend too much time of your life learning rubik's cube algorithms. So go with 12 at a time.

Reviewer: ```suggestion
The ZBLL sheets organize algorithms in groups of 12, by their corner permutation. We recommend learning each group of 12 at the same time. Learning too few algorithms at once hinders your 'Case-Alg' connection, and learning too many at once can be overwhelming.
```

You can set 2 kinds of these goals: Either a set time, so sub 3, or for advanced solvers, sub 2 to low 2, or simply aim for a time that is below 3rd of your average solve time, since LL takes about this much in solves.

> Tip: It's recommended to not practice all of the 12 algs at once. Instead, learn them in halves (so 6 each) or even in thirds (4 each), that way you can practice each case more.

Reviewer: ```suggestion
Tip: When learning a group of 12 algs, we suggest practicing one half at a time, or even one third at a time. That way, you can practice each case more.
```
